pybamm/discretisations/discretisation.py

```diff
@@ -739,6 +741,9 @@ def process_dict(self, var_eqn_dict):
             Equations ({variable: equation} dict) to dicretise
             (can be model.rhs, model.algebraic, model.initial_conditions or
             model.variables)
+        ics : bool, optional
+            Whether the equations are initial conditions. If True, the equations are
```

Comment:
```suggestion
            Whether the equations have initial conditions. If True, the equations are
```

Reply:
This is meant to be `are`, this is the function that discretises the equations (rhs, algebraic, initial conditions) and there is a special case for initial conditions (reference only gets taken away from initial conditions)